.rhcicd/clowdapp-aggregator.yaml

```diff
@@ -10,9 +10,9 @@ objects:
     name: notifications-aggregator
     labels:
       app: notifications-aggregator
-  annotations: 
-    ignore-check.kube-linter.io/no-liveness-probe: "short lived cronjob doesn't need probes"
-    ignore-check.kube-linter.io/no-readiness-probe: "short lived cronjob doesn't need probes"
+    annotations: 
+      ignore-check.kube-linter.io/no-liveness-probe: "short lived cronjob doesn't need probes"
+      ignore-check.kube-linter.io/no-readiness-probe: "short lived cronjob doesn't need probes"
```

Comment:
Checking with my yaml rule and i think this looks fine
![Screenshot_2022-04-28_10-52-28](https://user-images.githubusercontent.com/3845764/165804298-37d6a871-12d7-480a-ade1-57ed0052189c.png)

Reply:
now i want a yaml ruler too